api/v1alpha/framework/xcuitest.schema.json

```diff
@@ -174,6 +174,14 @@
           },
           "smartRetry": {
             "$ref": "../subschema/common.schema.json#/definitions/smartRetry"
+          },
+          "shard": {
+            "description": "When shard is configured as concurrency, saucectl automatically splits the tests by concurrency so that they can easily run in parallel.",
+            "type": "string"
```

Comment:
Shouldn't this be an enum? After all, it only accepts two values right now:
- empty
- "concurrency"